domains/_vercel.whisp.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "itswhisp",
+    "email": "ag.whisp@gmail.com",
+    "discord": "1276688713434529854"
+  },
+  "record": {
+    "TXT": "vc-domain-verify=whisp.is-a.dev,131ed1253e922c1133f2"
```

Comment:
i think you need to do `"TXT": ["vc-domain-verify=whisp.is-a.dev,131ed1253e922c1133f2"]` this ,instead of `"TXT": "vc-domain-verify=whisp.is-a.dev,131ed1253e922c1133f2"` this